.dockerignore

```diff
@@ -1,3 +1,4 @@
+tests
```

Comment:
This change is again not directly related to this refactor, but helped speed up the Docker builds